src/Files.App/ViewModels/ShellViewModel.cs

```diff
@@ -1219,7 +1221,19 @@ await dispatcherQueue.EnqueueOrInvokeAsync(() =>
 									item.SyncStatusUI = CloudDriveSyncStatusUI.FromCloudDriveSyncStatus(syncStatus);
 									item.FileFRN = fileFRN;
 									item.FileTags = fileTag;
-									item.ContextualProperty = $"{Strings.Modified.GetLocalizedResource()}: {item.ItemDateModified}";
+									
+									if (itemType.Equals("Generic hierarchical", StringComparison.Ordinal) && extraProperties is not null && extraProperties.Result["System.Capacity"] is not null && extraProperties.Result["System.FreeSpace"] is not null)
```

Comment:
This `if` works only when system (not Files's one) language is set to English. `ItemType` is translated by Windows

Reply:
I was hoping @gave92 would know a better way to detect this 😂